collections/open-journalism/index.md

```diff
@@ -0,0 +1,6 @@
+---
+items: fivethirtyeight/data, NYTimes/objective-c-style-guide, newsapps/beeswithmachineguns, voxmedia/meme, propublica/guides, censusreporter/censusreporter, nprapps/app-template, TimeMagazine/babynames, guardian/frontend, dukechronicle/chronline, BloombergMedia/whatiscode, times/cardkit
```

Comment:
I'm curious why it needs to be a single line. Jekyll Front Matter supports lists with each item on a separate line, e.g.,:

```yaml
items:
- fivethirtyeight/data
- NYTimes/objective-c-style-guide
- newsapps/beeswithmachineguns
- voxmedia/meme
- propublica/guides
- censusreporter/censusreporter
- nprapps/app-template
- TimeMagazine/babynames
- guardian/frontend
- dukechronicle/chronline
- BloombergMedia/whatiscode
- times/cardkit
```

I find this format more readable, and it seems like it'd save some parsing trouble for you. That is, you wouldn't have to split on a comma, the normal Front Matter parser would generate a list for you.

Reply:
@cheshire137 @talsafran I will take a look at this